### ISSUE TITLE
sending client certificates

### ISSUE THREAD

**Issue by moley** [rightscale/wstunnel#41]
Hello, 

I have configured wstunnel to use https on serverside, like in example with an nginx. 
I'd like to initialize an wstunnel from client and want to send some client certificates, which should do authentication on server side. 
Is there yet a possibility to do this?

**Comment by moley** [comment]
Any ideas?

**Comment by rshade** [comment]
@moley I don't think so, I would have to check the underlying libraries. Would you be willing to do a PR?
